worlds/sc2/test/test_usecases.py

```diff
@@ -279,6 +279,6 @@ def test_race_swap_pick_one_has_correct_length_and_includes_swaps(self) -> None:
         self.generate_world(world_options)
         world_regions = [region.name for region in self.multiworld.regions]
         world_regions.remove('Menu')
-        self.assertEqual(len(world_regions), len(vanilla_mission_req_table.get(SC2Campaign.WOL)))
+        self.assertEqual(len(world_regions), 25) # Number of missions in WoL without Prophecy

```

Comment:
Rather than a magic number, may be better to derive this from `SC2Mission`?

`NUM_WOL_MISSIONS = len([mission for mission in SC2Mission if mission.campaign == SC2Campaign.WOL and MissionFlag.RaceSwap not in mission.flags])`